src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 7) {
+    return 230;
+  }
+
+  if (days === 6) {
+    return 220;
+  }
+
+  if (days === 3) {
+    return 100;
+  }
+
+  return 80;
```

Comment:
This return statement does not account for rentals of less than 3 days. The function should calculate the total cost based on the daily rate multiplied by the number of days rented.